_pages/en_US/include/menuhax67-install.txt

```diff
@@ -0,0 +1,6 @@
+1. Press (Y) + (D-Pad Down) to install menuhax67
+  * The console will automatically power off
+1. Power on your console, then tap the HOME Menu Settings icon on the top-left corner of the screen to open the Homebrew Launcher
```

Comment:
split `power on your console` and the rest of the steps into two steps, just to make translating life easier even a little bit